quinn-proto/src/config.rs

```diff
@@ -237,8 +225,8 @@ impl Default for TransportConfig {
         const STREAM_RWND: u32 = MAX_STREAM_BANDWIDTH / 1000 * EXPECTED_RTT;
 
         TransportConfig {
-            stream_window_bidi: 32u32.into(),
-            stream_window_uni: 32u32.into(),
+            max_concurrent_bidi_streams: 1024u32.into(),
```

Comment:
1024 is rather high. The config allows a receive buffer size of 1.25MB per stream -> This allows for a memory usage of more than 1GB on reception, in case the application can't keep up (it hopefully can, since the whole thing is limited by the speed of a single connection).

I nevertheless think something between the old 32 and maybe around 100 would make for a reasonable default config.

Reply:
Good catch, didn't really think that one through.